src/test/java/com/znsio/teswiz/screen/android/ScreenShotScreenAndroid.java

```diff
@@ -18,6 +20,9 @@ public ScreenShotScreenAndroid(Driver driver, Visual visually) {
     @Override
     public ScreenShotScreen takeScreenshot() {
         visually.checkWindow(SCREEN_NAME, "Take Screenshot");
+        driver.getInnerDriver().get("https://github.com/znsio/teswiz");
+        waitFor(3);
```

Comment:
We are not supposed to add wait.

you can use waittillelementis present

Reply:
Reverted file